lib/analyzer/docker-analyzer.ts

```diff
@@ -19,6 +19,15 @@ export async function analyze(
   dockerfileAnalysis?: dockerFile.DockerFileAnalysis,
   options?: DockerOptions,
 ) {
+  await imageInspector.pullIfNotLocal(targetImage, options).catch((err) => {
```

Comment:
why do you use `.catch` instead of wrapping with regular try-catch? (in the original PR the error was `No such object`)

Reply:
Actually only did that because that's used in the rest of the file